test/unit/services/subscription/CdsSubscriberSpec.scala

```diff
@@ -159,14 +159,6 @@ class CdsSubscriberSpec extends UnitSpec with MockitoSugar with ScalaFutures wit
             Some(emailVerificationTimestamp)
           )
           inOrder.verify(mockCdsFrontendDataCache).registerWithEoriAndIdResponse(any[Request[AnyContent]])
-          inOrder
```

Comment:
Is there a reason why we have moved this? Let me know if better to discuss on a call.

Reply:
This caused a test failure when we changed the sub02outcome to not redirect once refreshing the page